client/webserver/site/cache_utilities.bash

```diff
@@ -0,0 +1,15 @@
+SITE_DIR=`cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P`
+BB_FILEPATH=${SITE_DIR}/src/html/bodybuilder.tmpl
+CSS_DIR=${SITE_DIR}/src/css
+JS_DIR=${SITE_DIR}/src/js
+
+bbhash () { sha1sum ${SITE_DIR}/src/html/bodybuilder.tmpl | head -n1 | cut -d " " -f1 ; }
+hashdir () { sha1sum $1/* | md5sum | head -n1 | cut -d " " -f1 ; }
```

Comment:
I guess this hashing scheme isn't going to work, because tests are passing locally but not in CI. Probably some small difference in the output of `sha1sum` when reading a directory. I initially had something like
```bash
git ls-files -s ${CSS_DIR} | git hash-object --stdin
```
but `ls-files -s` prints the commited hash, not the current hash, so you would be have to commit, generate cache busters, and then amend, which is crap. Also no guarantee that git versions are consistent with that `ls-files` directory either. 

Maybe this won't work after all.

Reply:
I think it's because we have https://github.com/decred/dcrdex/blob/5daece26338af5c056e4ed00ee282973c00c300b/client/webserver/site/webpack/common.js#L41-L43
so it turns out to be the annoying process like you said.

We could remove that from the webpack build and then the content hash wouldn't change with the git commit advancing.